docs/enterprise-solutions/configuration/infrastructure-configuration/control-other-cline-features/yolo-mode.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Yolo Mode
+description: Configure Yolo Mode settings for enterprise environments
+---
+
+## Overview
+
+Yolo Mode enables Cline to operate with increased automation while maintaining enterprise security and compliance standards. When enabled, Cline can execute approved operations with minimal user confirmation, accelerating development workflows.
```

Comment:
Would be great to crossreference to the original Yolo Mode docs in the main docs page so people can get all the details of Yolo Mode.